tools/python_api/src_py/async_connection.py

```diff
@@ -176,6 +176,8 @@ async def execute(
 
     async def _prepare(self, query: str, parameters: dict[str, Any] | None = None) -> PreparedStatement:
         """
+        Prepare a query.
+
```

Comment:
This method is intentionally not commented. It is not considered as an internal method and not intended to be used by the user.